frontend/package.json

```diff
@@ -155,6 +155,7 @@
     "type-check": "tsc --noEmit",
     "lint": "concurrently \"node_modules/.bin/next lint\" \"node_modules/.bin/stylelint '**/*.{js,ts,tsx,css}'\"  \"npm run type-check\"",
     "prettier-check": "npx prettier --check .",
+    "prettier-fix": "npx prettier --write .",
```

Comment:
thanks for adding this! I think most of the time VSCode fixes it for me on save 😆